src/core/MOM_open_boundary.F90

```diff
@@ -3957,7 +3977,8 @@ subroutine update_OBC_segment_data(G, GV, US, OBC, tv, h, Time)
 
     if (.not. segment%on_pe) cycle ! continue to next segment if not in computational domain
 
-    ! NOTE: These are in segment%HI, but defined slightly differently
+    ! NOTE: segment%is_obc and segment%ie_obc are range of indices for the full segment.
+    !  The other date set here are in segment%HI, but here they defined slightly differently.
```

Comment:
Do you mean "data" instead of date?

Reply:
Yes, I did.  Thank you for spotting this.